integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/standalone/CcWithDefaultConf.java

```diff
@@ -23,6 +23,7 @@
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
+import org.wso2.choreo.connect.discovery.api.Api;
```

Comment:
Is this intentional? looks this import is the only change in this file

Reply:
Thank you, it is a mistake. I will remove it.